packages/lib-content/src/components/Article/Article.js

```diff
@@ -33,11 +37,17 @@ export default function Article({
       {imageSrc.length && size !== 'small' ? (
         <Box
           alignSelf='center'
-          background={`url('${imageSrc}')`}
           flex='grow'
-          height={{ max: '120px', min: '120px' }}
-          width={{ max: '120px', min: '120px' }}
-        />
+          height={thumbnailSize}
+          width={thumbnailSize}
+        >
+          <Image
+            fit='cover'
+            src={imageSrc}
+            alt={title}
+            loading='lazy'
```

Comment:
Not listed in Grommet's docs but does it pasted on to the html element ([loading](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading)) 👍